include/zephyr/bluetooth/audio/csis.h

```diff
@@ -50,6 +50,9 @@ extern "C" {
 /** Size of the Set Identification Resolving Key (SIRK) */
 #define BT_CSIS_SET_SIRK_SIZE 16
 
+/** Size of the Resolvable Set Identifier (RSI) */
+#define BT_CSIS_RSI_SIZE                        6
```

Comment:
Minor: It probably would have been preferred if the renaming of PRSI -> RSI was in a separate commit

Reply:
I'll remember that.  Required now?